lib/index.js

```diff
@@ -1,23 +1,17 @@
-const fs = require('fs')
 const SlskClient = require('./slsk-client')
 let stack = require('./stack')
 
-let client
-
-const mkdirSync = function (dirPath) {
-  try {
-    fs.mkdirSync(dirPath)
-  } catch (err) {
-    if (err.code !== 'EEXIST') throw err
-  }
-}
+let client = null
+let tmpDir = null
 
 module.exports = {
+  tmpDir,
   connect: (obj, cb) => {
     if (typeof cb !== 'function') {
       throw new Error('2nd argument must be callback function')
     }
-    mkdirSync('/tmp/slsk')
+
+    obj.tmpDir = tmpDir || '/tmp/slsk'
```

Comment:
Hi! You should use `os.tmpdir()` instead of `/tmp`, so `path.join(os.tmpdir(), 'slsk')`